covid19_sfbayarea/data/san_mateo/deaths_by_ethnicity.py

```diff
@@ -0,0 +1,13 @@
+from typing import Dict, List
+from .power_bi_querier import PowerBiQuerier
+
+class DeathsByEthnicity(PowerBiQuerier):
+    def __init__(self) -> None:
+        self.source = 'd'
+        self.name = 'deaths by race'
+        self.property = 'race'
+        super().__init__()
+
+    def _parse_data(self, response_json: Dict[str, List]) -> List[Dict[str, int]]:
+        results = super()._parse_data(response_json)
+        return [ { ethnicity.strip(): count } for ethnicity, count in results ]
```

Comment:
can you also do a `map_ethnicity_to_data_model`, like you did up in `casesbyethnicity.py`?

Reply:
Actually, please hold off for now because now I'm feeling strongly about reporting the race/eth labels just as the county does! We'll see how the poll goes. Sorry to introduce more chaos.

Reply:
Hey, no worries. I've already done this locally. It's no big deal to commit it or revert it. Do you have a preferred way on this?

I'd rather either do it for both or neither.

Reply:
For the record, I requested that race/eth labels be reported just as the county does.